content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -123,6 +123,32 @@ const prisma = new PrismaClient({
 })
 ```
 
+### <inlinecode>adapter</inlinecode>
+
+Defines an instance of a driver adapter. See also [Drivers](/concepts/components/drivers) <span class="concept"></span>.
+
+This is available from version 5.4.0 and upwards. 
+
+This feature is currently in Preview behind the `driverAdapters` feature flag.
+
+#### Example
+
+```ts
+import { Pool, neonConfig } from '@neondatabase/serverless'
+import { PrismaNeon } from '@prisma/adapter-neon'
+import { PrismaClient } from '@prisma/client'
+import dotenv from 'dotenv'
+import ws from 'ws'
+
+dotenv.config()
+neonConfig.webSocketConstructor = ws
+const connectionString = `${process.env.DATABASE_URL}`
+
+const pool = new Pool({ connectionString })
+const adapter = new PrismaNeon(pool)
+const prisma = new PrismaClient({ adapter })
```

Comment:
Worth making an abstract "adapter" here instead of using the Neon example? This adds more confusion than it helps here if you have no prior context about the neon driver, driver adapters and so on.
```suggestion
import { driver } from '@example/driver'
import { PrismaExampleAdapter } from '@prisma/adapter-example'
import { PrismaClient } from '@prisma/client'

const connectionString = `${process.env.DATABASE_URL}`

const driver = new driver({ connectionString })
const adapter = new PrismaExample(driver)
const prisma = new PrismaClient({ adapter })
```

Reply:
I think it would make sense to have an example adapter that works rather than an abstract one.

I would be happy to add a note highlighting that the example uses the Neon adapter specifically.

Reply:
Then do that.